src/python/pants/engine/legacy/graph.py

```diff
@@ -399,14 +400,10 @@ class HydratedTargets(Collection.of(HydratedTarget)):
 
 
 class OwnersRequest(datatype([
-  ('sources', tuple),
-  ('include_dependees', str),
+  ('sources', TypedCollection(Exactly(text_type))),
```

Comment:
Could you separate out and PR the "changing un-typed fields to typed/enum fields" changes?

Reply:
Will do!

Reply:
Made into #7364!